### PR TITLE
Form to epost a prescription

### PR DESCRIPTION
**Pull request by alex-blair**
[Jira Ticket](https://caracare.atlassian.net/browse/CCA-2218)

## Description

Adds a new page to the Dashboard with a form that allows a user to upload a PDF prescription and send it to the backend, where it will be forwarded to the Deutsch Post ePost service. 

### New section on Dashboard

image

#### Form for ePost service

image

#### Success modal

image

## Checklist
- [x] Lint and unit tests pass locally with my changes.

### CODE REVIEW

File: src/epost/PrescriptionEpostService.tsx [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+            </Typography>
+            <input
+              type="file"
+              name="upload"
+              accept=".pdf"
+              className={styles.fileInput}
+              ref={fileInput}
+              onChange={handleFileInputChange}
+            />
+            {!!pdfName ? (
```
Comment (drmas) [inline src/epost/PrescriptionEpostService.tsx:303]: Why do we use separate state, we can use `formData.pdfFile?.name`

File: src/epost/PrescriptionEpostService.tsx [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+      const ePostFormData = formatFormData({
+        ...formData,
+        pdfFile: formData.pdfFile,
+      });
+      sendPrescription(ePostFormData)
+        .then((res: any) => {
+          setShowSuccessMessage(true);
+          setError('');
+          setShowSuccessMessage(true);
+        })
```
Comment (drmas) [inline src/epost/PrescriptionEpostService.tsx:130]:
```suggestion
          setShowSuccessMessage(true);
          setError('');
        })
```

**Review by drmas — COMMENTED** [review]
Nice work 👏  , 2 small comments

File: src/epost/PrescriptionEpostService.tsx (hunk 1 above)
Comment (alex-blair) [reply src/epost/PrescriptionEpostService.tsx:302]: Good point - I'll update!

**Comment by drmas** [comment]
@alex-blair  Can you check why CI failures

**Comment by alex-blair** [comment]
@drmas CI is working again

**Review by drmas — APPROVED** [review]

**Pull request merged**